packages/api/tests/endpoints/providers/create-provider.js

```diff
@@ -196,3 +196,26 @@ test('POST returns a 400 response if invalid hostname is provided', async (t) =>
     .expect(400);
   t.is(response.status, 400);
 });
+
+test('CUMULUS-176 POST returns a 404 if ', async (t) => {
```

Comment:
Need to update this test description

Reply:
Fixed here: 507ab283261afbc3c61cb9d507a8ecff52ce41ff